sections/kubecontext.zsh

```diff
@@ -29,6 +29,10 @@ spaceship_kubecontext() {
 
   spaceship::exists kubectl || return
 
+  # if kubectl can't connect kubernetes cluster, kubecontext section will be not shown
+  local kubectl_version=$(kubectl version --short 2>/dev/null | grep "Server Version" | awk '{ match($0, /^Server Version: (.+)$/, version); print version[1] }')
+  [[ -z $kubectl_version ]] && return
```

Comment:
Could you pls explain why this fixes the issue? `kubectl config current-context` doesn't seem to depend on whether remote cluster is accessible.

I'm also a bit worried that running `kubectl version` will drastically slow down segment rendering, in cases when it actually tries to connect to the cluster over a slow connection.

Reply:
See #748 . When I can not connect to the cluster, it is strange to show kubecontext

Reply:
I've seen the issue but I don't understand root cause, all those guys have kubernetes cluster configured but they currently have no internet and therefore cannot connect to it, and they ask why kubernetes segment is still visible?